npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
The same logic exists here: https://github.com/stealjs/system-npm/blob/master/npm-convert.js#L279

I'd put your transform function in npm-utils, maybe call it jsonTransform, so it can be used in both places.